recipes/astro-gala/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "astro-gala" %}
+{% set version = "0.2.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "83b319d1e05dbe0453671a3d28cadacb7f57bcf17e3f7e24441410b4cd1ff134" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
```

Comment:
``--offline`` was needed over there in https://github.com/astropy/conda-channel-astropy for all packages that relied on the astropy_helpers submodule.